jubatus/server/server/classifier.idl

```diff
@@ -59,7 +59,7 @@ service classifier {
   #-
   #-  Get all labels in the model
   #@random #@nolock #@pass
-  list<string> get_labels()
+  map<string, int> get_labels()
```

Comment:
Can't we use `ulong` (uint64_t)? `int` (int32_t) Sounds insufficient for linear classifiers.